docs/csharp/language-reference/builtin-types/floating-point-numeric-types.md

```diff
@@ -0,0 +1,99 @@
+
+The following table shows the precision and approximate ranges for the floating-point types.  
+  
+|Type|Approximate range|Precision|  
+|----------|-----------------------|---------------|  
+|`float`|±1.5 x 10<sup>−45</sup> to ±3.4 x 10<sup>38</sup>|~6-9 digits|  
+|`double`|±5.0 × 10<sup>−324</sup> to ±1.7 × 10<sup>308</sup>|~15-17 digits|  
+|`decimal`|±1.0 x 10<sup>-28</sup> to ±7.9228 x 10<sup>28</sup>|28-29 digits|  
+
+The default value for all floating-point types is `0`. Each of the floating-point types has constants named `MinValue` and `MaxValue` for the minimum and maximum value for that type. The `float` and `double` types have additional constants for `PositiveInfinity`, `NegativeInfinity`, and `NaN` (for "Not a Number"). The `decimal` type includes constants for `Zero`, `One`, and `MinusOne`.
+
+The `decimal` type has more precision and a smaller range than both `float` and `double`, which makes it appropriate for financial and monetary calculations.
+
+You can mix numeric integral types and floating-point types in an expression. In this case, the integral types are converted to floating-point types. The evaluation of the expression is performed according to the following rules:
+
+- If one of the floating-point types is `double`, the expression evaluates to `double`, or to [bool](../keywords/bool.md) in relational comparisons or comparisons for equality.
+- If there is no `double` type in the expression, the expression evaluates to `float`, or to [bool](../keywords/bool.md) in relational comparisons or comparisons for equality.
+
+A floating-point expression can contain the following sets of values:
+
+- Positive and negative zero
+- Positive and negative infinity
+- Not-a-Number value (NaN)
+- The finite set of nonzero values
+
+For more information about these values, see IEEE Standard for Binary Floating-Point Arithmetic, available on the [IEEE](https://www.ieee.org) website.
+
+## floating point literals
+
+By default, a real numeric literal on the right side of the assignment operator is treated as `double`. You can use suffixes to convert a literal to a specific type. 
+
+- The `d` or `D` suffix converts an integral literal to a `double`.
+- The `f` or `F` suffix converts a real or integral literal to a `float`.
+- The `m` or `M` suffix converts a real or integral literal to a `decimal`.
+
+The following examples show each suffix:
+
+```csharp
+double d = 3D;
+d = 4d;
+float f = 3.5F;
+f = 5.4f;
+decimal myMoney = 300.5m;
+myMoney = 400.75M;
+```
+
+## Conversions
+
+There's an implicit conversion (called a *widening conversion*) between any two floating-point types where the destination type can store all values of the source type. For example, there's an implicit conversion from `float` to `double` because the range of `float` values is a proper subset of `double`. 
```

Comment:
This paragraph gives impression that several widening conversions exists. However, there is only one widening conversion (between floating-point types): from `float` to `double`.

Reply:
Thanks. I was torn between matching the description for integral types, and the fact that there is only one.